index.bs

```diff
@@ -1716,16 +1719,35 @@ associated.
 </xmp>
 <div dfn-type="dict-member" dfn-for="PublicKeyCredentialEntity">
     :   <dfn>name</dfn>
-    ::  A human-readable name for the entity. Its function depends on what the {{PublicKeyCredentialEntity}} represents:
+    ::  A [=human palatability|human-palatable=] name for the entity. Its function depends on what the {{PublicKeyCredentialEntity}} represents:
 
-          - When inherited by {{PublicKeyCredentialRpEntity}} it is a human-friendly identifier for the [=[RP]=], intended only
+          - When inherited by {{PublicKeyCredentialRpEntity}} it is a [=human palatability|human-palatable=] identifier for the [=[RP]=], intended only
             for display. For example, "ACME Corporation", "Wonderful Widgets, Inc." or "ОАО Примертех".
+
+              - [=[RPS]=] SHOULD perform preparation and enforcement, as prescribed in [[!RFC8266]]
+                Sections 2.2 and 2.3 for the Nickname Profile of the PRECIS FreeformClass [[!RFC8264]],
+                when setting {{PublicKeyCredentialEntity/name}}'s value, or displaying the value to the user.
+
+              - [=Clients=] MUST perform preparation and enforcement, as prescribed in [[!RFC8266]]
+                Sections 2.2 and 2.3 for the Nickname Profile of the PRECIS FreeformClass [[!RFC8264]],
+                on {{PublicKeyCredentialEntity/name}}'s value prior to displaying the value to the user or
+                including the value as a parameter of the [=authenticatorMakeCredential=] operation.
```

Comment:
Is this a breaking normative change?

Reply:
Looks like it to me...

Reply:
BTW, I would just say "enforcement" (not also "preparation") because it's enforcement that really matters here.

Reply:
I'm not against a MUST in and of itself, but I expect @nadalin and/or @akshayku will object to a breaking change...

Reply:
@stpeter 
> I would just say "enforcement" (not also "preparation")

thx, yeah, also it turns out "enforcement" invokes "preparation".  I corrected this in https://github.com/w3c/webauthn/pull/951/commits/3ec508d155a3e84a4f8253961282dd1fa361299c